kong/plugins/grpc-gateway/deco.lua

```diff
@@ -199,6 +200,34 @@ local function unframe(body)
   return body:sub(pos, frame_end), body:sub(frame_end + 1)
 end
 
+--[[
+  // Set value `v` at `path` in table `t`
+  // Path contains value address in dot-syntax. For example:
+  // path="a.b.c" would lead to `t[a][b][c] = c`
+]]
+local function add_to_table( t, path, v )
+  local function patch_table( t, path, v )
```

Comment:
this function doesn't use any upvalue from `add_to_table()`.  can it be moved out of that function?  otherwise the (empty) closure object is created on each call.

Reply:
I was afraid of spoiling global namespace with `patch_table` helper. But sure, it is outside now.